docs/proposals/004-endpoint-picker-protocol/README.md

```diff
@@ -57,6 +63,25 @@ Constraints:
 - The EPP MUST not set two different values in the header and the inner response metadata value.
 - Setting different value leads to unpredictable behavior because proxies aren't guaranteed to support both paths, and so this protocol does not define what takes precedence.
 
+## Destination Endpoint Status
+
+[RESPONSE: Data Plane -> EPP]
+
+For each HTTP response, the data plane MUST communicate to the EPP endpoint status as follows:
+
+In the ext_proc [ProcessingRequest.metadata_context](https://github.com/envoyproxy/envoy/blob/v1.35.0/api/envoy/service/ext_proc/v3/external_processor.proto#L130) field, a new metadata field "x-gateway-destination-endpoint-status" is added as an unstructured entry with an outer key (which represents the metadata namespace) with a default value of `envoy.lb`:
+
+```go
+filterMetadata: {
+  "envoy.lb": {
+    "x-gateway-destination-endpoint-status": endpoint=<ip:port>,
```

Comment:
not sure the name `x-gateway-destination-endpoint-status` is so clear.
maybe a more descriptive name like `x-gateway-served-by-endpoint`?

Reply:
I've changed the name. PTAL.